### PR TITLE
text.low-cased

### PR DESCRIPTION
**Pull request by EugeneDar**
https://github.com/objectionary/eo-strings/issues/76

### CODE REVIEW

**Comment by EugeneDar** [comment]
@yegor256 @Graur please have a look

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -172,7 +172,37 @@
   # @todo #57:30min Current implementation using Java.
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/text.eo:172]: @EugeneDar looks like you gorget to remove this `todo`

**Review by Graur — CHANGES_REQUESTED** [review]

**Comment by EugeneDar** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merg

**Comment by EugeneDar** [comment]
@yegor256 can you merge it again?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**